NEWS.md

```diff
@@ -8,13 +8,14 @@
 - Add section to theme generation that will use theme-specific color(s) to set colors of vim/emacs-mode cursor
 
 #### Posit Workbench
-- Prometheus metrics are available in Posit Workbench (rstudio-pro:#3273)
-- Additional support for publishing new content types to Posit Cloud (rstudio-pro#4541)
-- Update code-server to version 4.14.1 (VS Code version 1.79.2) bundled with node 16.20.2 (rstudio-pro#4774, rstudio-pro#5012)
-- Update rserver-saml to 0.7.0 (rstudio-pro#4903)
-- Change the default behavior of `launcher-balancing-enabled` to always be true. (rstudio-pro#4953)
-- Auto-generate the `secure-cookie-key` and launcher key pair in load-balanced environments (rstudio-pro#4985)
-- Add the `secure-cookie-key` and launcher key pair to the node table of the database for load-balanced environments. (rstudio-pro#4985)
+- Prometheus metrics are available in Posit Workbench (rstudio/rstudio-pro#3273)
+- Additional support for publishing new content types to Posit Cloud (rstudio/rstudio-pro#4541)
+- Update code-server to version 4.14.1 (VS Code version 1.79.2) bundled with node 16.20.2 (rstudio/rstudio-pro#4774, rstudio/rstudio-pro#5012)
+- Update rserver-saml to 0.7.0 (rstudio/rstudio-pro#4903)
+- Change the default behavior of `launcher-balancing-enabled` to always be true. (rstudio/rstudio-pro#4953)
+- Auto-generate the `secure-cookie-key` and launcher key pair in load-balanced environments (rstudio/rstudio-pro#4985)
+- Add the `secure-cookie-key` and launcher key pair to the node table of the database for load-balanced environments. (rstudio/rstudio-pro#4985)
+- Update VS Code Workbench extension to 1.4.7 (rstudio/rstudio-workbench-vscode-ext#227)
```

Comment:
There isn't any way for customers to see what's included in a specific version of the extension, so usually we described just changed in the extension rather than listing a release number. (Ideally NEWS.md would be updated in `rstudio-workbench-vscode-ext` and then we'd copy that content here)

Reply:
Got it